distributions/otelcol-ebpf-profiler/manifest.yaml

```diff
@@ -0,0 +1,37 @@
+dist:
+  module: github.com/open-telemetry/opentelemetry-collector-releases/ebpf-profiler
+  name: otelcol-ebpf-profiler
+  description: OpenTelemetry Collector for eBPF Profiling
+  version: 0.123.1
+  output_path: ./_build
+
+exporters:
+  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.123.0
+  - gomod: go.opentelemetry.io/collector/exporter/nopexporter v0.123.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.123.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.123.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.123.0
+
+processors:
+  - gomod: go.opentelemetry.io/collector/processor/batchprocessor v0.123.0
+  - gomod: go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.123.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sattributesprocessor v0.123.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.123.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourceprocessor v0.123.0
+
+receivers:
+  - gomod: go.opentelemetry.io/ebpf-profiler v0.0.0-20250408174414-ead430a15d49
```

Comment:
This doesn't look like it comes from Core or Contrib, which breaks one of the criteria